descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "item-storage",
-      "version": "8.9",
+      "version": "9.0",
```

Comment:
Why is this change considered a breaking compatibility change?

Given that it is considered as such, please: 
* ensure that this work is not merged until pull requests can be coordinated for all dependent modules
* the RAML files are also updated so the versions correspond

Reply:
It was decided that enabling "failOnConflict" is a breaking change with major interface bump: https://github.com/folio-org/raml-module-builder#api-and-schema-versioning-for-optimisticlocking
The corresponding mod-inventory changes are in https://issues.folio.org/browse/MODINV-398 = https://github.com/folio-org/mod-inventory/pull/344

Reply:
All RAML files have corresponding versions.